src/get_clades_to_model.py

```diff
@@ -19,6 +19,7 @@
 """
 
 # /// script
+# requires-python = ">=3.11"
```

Comment:
Is this some UV-specific incantation that avoids the usual requirements.txt and .python-version?